Tests/EastAsianWidthTests/EdgeCharacters+Generated.swift

```diff
@@ -11,35 +11,35 @@ let EastAsianFullwidthEdgeRawScalarValue: [UInt32] = [
     0x3000,0xff01,0xff60,0xffe0,0xffe6,
 ]
 
-let EastAsianFullwidthEdgeUnicodeScalar: [UnicodeScalar] = EastAsianFullwidthEdgeRawScalarValue.flatMap(UnicodeScalar.init)
+let EastAsianFullwidthEdgeUnicodeScalar: [UnicodeScalar] = EastAsianFullwidthEdgeRawScalarValue.compactMap(UnicodeScalar.init)
```

Comment:
If you want to modify them, please modify line which `flatMap` in this script code

https://github.com/HevaWu/EastAsianWidth/blob/b70ddd0501b41e6c5dca1cc2c94de5ca315b4698/Scripts/TestGenerator.py#L28

Reply:
oh no.

I don't think it's a good way to generate everyithing with python